content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Pods And Endpoints Termination Flow
+content_type: tutorial
+weight: 60
+---
+
+
+<!-- overview -->
+
+## The Kubernetes model for connecting containers
+
+Now that you have a continuously running, replicated application, that is
+exposed on a network, let's look at Pods termination flow and ways to implement
+a graceful connections draining.
+
+There are often a need to terminate a Pod - be it for upgrade or scale down.
```

Comment:
Grammatical nit. Also we should also be explicit about what is being upgraded/scaled down.
```suggestion
There are often cases when you need to terminate a Pod - be it for upgrade or scale down.
```